lib/bootloader_setup.pm

```diff
@@ -448,6 +448,16 @@ sub bootmenu_network_source {
                 # As we use boot options, no extra action is required
                 return;
             }
+
+            # fate#322276
+            # Point linuxrc to a repomd repository and load the installation system from local medium
+            if (($m_protocol eq "http") && (get_var('REMOTE_REPOINST'))) {
```

Comment:
Could you pleas add description to the variables.md?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6355